### DISCUSSION TITLE
Advice on using chart library for console

### DISCUSSION THREAD

**Discussion by cschandragiri** [twilio-labs/paste#1051] [General] [answered]
I see the [react components for Charts ](https://code.hq.twilio.com/pages/react/react-components/#/UI%20Components/Chart) is deprecated and we don't have a chart component in [paste](https://paste.twilio.design/) yet. When shall chart components be available in paste? Any risk in using the deprecated component. Can we use the [highchart-react component](https://github.com/highcharts/highcharts-react#highcharts-with-typescript) directly? 

Our new design for debugger using Charts: https://twilio.invisionapp.com/share/NUZLGG8W9BG#/screens/439248474

**Comment by ghostling** [comment, marked as answer]
We had a similar use case in iot-console earlier this year, and decided to use highcharts directly while we wait for something similar in Paste (if that's on the roadmap..). I wouldn't recommend using the react-components library for this as those dependencies are quite old and it's harder to find/navigate older documentation.

**Reply by corinne-nagel** [reply]
Thanks for your reply @ghostling !

**Reply by cschandragiri** [reply]
Thanks @ghostling
